Formula/l/llamafile.rb

```diff
@@ -0,0 +1,43 @@
+class Llamafile < Formula
+  desc "Distribute and run LLMs with a single file"
+  homepage "https://github.com/Mozilla-Ocho/llamafile"
+  url "https://github.com/Mozilla-Ocho/llamafile/archive/refs/tags/0.6.tar.gz"
+  sha256 "9ac8449268287c45fb015f7068b223be7336c6d7f8170f3adb9a370b65e8b9f0"
+  license "Apache-2.0"
+  head "https://github.com/Mozilla-Ocho/llamafile.git", branch: "main"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "make"
+
+  skip_clean "bin/llamafile",
+             "bin/llamafile-convert",
+             "bin/llamafile-perplexity",
+             "bin/llamafile-quantize",
+             "bin/llava-quantize",
+             "bin/zipalign"
```

Comment:
Why?

Reply:
Without this the clean step clears the exec bit.

Reply:
That's the first time I've ever heard of that happening. Can you remove it and let the test check that

Reply:
Without this I see the clean step dropping the exec permission:

```console
$ HOMEBREW_NO_INSTALL_FROM_API=1 brew install --debug --verbose --build-from-source llamafile
...
install -m 0644 llama.cpp/llava/llava-quantize.1 /opt/homebrew/Cellar/llamafile/0.6.1/share/man/man1/llava-quantize.1
/usr/bin/env /opt/homebrew/Library/Homebrew/shims/shared/git --version
==> Cleaning
==> Fixing /opt/homebrew/Cellar/llamafile/0.6.1/bin/llamafile permissions from 755 to 444
==> Fixing /opt/homebrew/Cellar/llamafile/0.6.1/bin/llamafile-convert permissions from 755 to 555
==> Fixing /opt/homebrew/Cellar/llamafile/0.6.1/bin/llamafile-perplexity permissions from 755 to 444
==> Fixing /opt/homebrew/Cellar/llamafile/0.6.1/bin/llamafile-quantize permissions from 755 to 444
==> Fixing /opt/homebrew/Cellar/llamafile/0.6.1/bin/llava-quantize permissions from 755 to 444
==> Fixing /opt/homebrew/Cellar/llamafile/0.6.1/bin/zipalign permissions from 755 to 444
==> Finishing up
/opt/homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromDefaultTapPathLoader): loading llamafile
ln -s ../Cellar/llamafile/0.6.1/bin/llamafile llamafile
ln -s ../Cellar/llamafile/0.6.1/bin/llamafile-convert llamafile-convert
ln -s ../Cellar/llamafile/0.6.1/bin/llamafile-perplexity llamafile-perplexity
ln -s ../Cellar/llamafile/0.6.1/bin/llamafile-quantize llamafile-quantize
ln -s ../Cellar/llamafile/0.6.1/bin/llava-quantize llava-quantize
ln -s ../Cellar/llamafile/0.6.1/bin/zipalign zipalign
ln -s ../../../Cellar/llamafile/0.6.1/share/man/man1/llamafile-perplexity.1 llamafile-perplexity.1
ln -s ../../../Cellar/llamafile/0.6.1/share/man/man1/llamafile-quantize.1 llamafile-quantize.1
ln -s ../../../Cellar/llamafile/0.6.1/share/man/man1/llamafile.1 llamafile.1
ln -s ../../../Cellar/llamafile/0.6.1/share/man/man1/llava-quantize.1 llava-quantize.1
ln -s ../../../Cellar/llamafile/0.6.1/share/man/man1/zipalign.1 zipalign.1
==> Summary
🍺  /opt/homebrew/Cellar/llamafile/0.6.1: 15 files, 18.7MB, built in 1 minute 8 seconds
==> Running `brew cleanup llamafile`...
...
$ ls -lh /opt/homebrew/bin/llamafile* 
lrwxr-xr-x  1 gguthe  admin    39B Jan 22 12:36 /opt/homebrew/bin/llamafile -> ../Cellar/llamafile/0.6.1/bin/llamafile
lrwxr-xr-x  1 gguthe  admin    47B Jan 22 12:36 /opt/homebrew/bin/llamafile-convert -> ../Cellar/llamafile/0.6.1/bin/llamafile-convert
lrwxr-xr-x  1 gguthe  admin    50B Jan 22 12:36 /opt/homebrew/bin/llamafile-perplexity -> ../Cellar/llamafile/0.6.1/bin/llamafile-perplexity
lrwxr-xr-x  1 gguthe  admin    48B Jan 22 12:36 /opt/homebrew/bin/llamafile-quantize -> ../Cellar/llamafile/0.6.1/bin/llamafile-quantize
$ ls -lh /opt/homebrew/Cellar/llamafile/0.6.1/bin/llamafile*                                  
-r--r--r--  1 gguthe  admin   6.2M Jan 22 12:36 /opt/homebrew/Cellar/llamafile/0.6.1/bin/llamafile
-r-xr-xr-x  1 gguthe  admin   1.4K Jan 22 12:36 /opt/homebrew/Cellar/llamafile/0.6.1/bin/llamafile-convert
-r--r--r--  1 gguthe  admin   4.4M Jan 22 12:36 /opt/homebrew/Cellar/llamafile/0.6.1/bin/llamafile-perplexity
-r--r--r--  1 gguthe  admin   4.0M Jan 22 12:36 /opt/homebrew/Cellar/llamafile/0.6.1/bin/llamafile-quantize
```

I'll update the list to drop `llamafile-convert` at least since that's being installed correctly.